packages/cli/src/Server.ts

```diff
@@ -159,6 +159,7 @@ import { ExecutionEntity } from './databases/entities/ExecutionEntity';
 import { SharedWorkflow } from './databases/entities/SharedWorkflow';
 import { AUTH_COOKIE_NAME, RESPONSE_ERROR_MESSAGES } from './constants';
 import { credentialsController } from './api/credentials.api';
+import { oauth2CredentialController } from './api/oauth2Credential.api';
```

Comment:
There is an extra `s` in the router name, triggering this and blocking build: `'"./api/oauth2Credential.api"' has no exported member named 'oauth2CredentialController'. Did you mean 'oauth2CredentialsController'?ts(2724)`